asm/bss_8001C4D0.s

```diff
@@ -2073,7 +2073,7 @@ glabel D_8018EDEB
 glabel D_8018EDEC
 .skip 1
 
-glabel D_8018EDED
+glabel gGameMenuSelectionDepth
```

Comment:
Depth is like: 1p,2p,3p,4p (depth of 1). VS, GP, Battle (depth of 2)? Yea, I think that's a good name.

Is `game` necessary? Or how about `MainMenu`?

Reply:
Yup, that's right.

`Game` matches the define I gave for this menu (`GAME_SELECT_MENU`), which was based on the text banner in the game saying 'Game Select'. Though I do think `MainMenu` is probably more intuitive. I can change this and the relevant #defines to `MainMenu`.